### PR TITLE
Feature/976

### PR DESCRIPTION
**Pull request by fraincs**
Issue: 

## Summary

Trying to override a Lozenge color and backgroundColor was throwing a typescript error as well as not working in some situations (when a lozenge had the highlight prop).

## What I did

Removed the typescript omissions in the component declaration. Modified the CSS of lozenge to work when highlighted.

## How to test

/?path=/story/chromatic-lozenge--inherit-parent-text-properties and by adding a color and backgroundColor prop to a lozenge

### CODE REVIEW

File: packages/components/src/lozenge/src/Lozenge.css [hunk 1]
```diff
@@ -54,7 +54,7 @@
 }
 
 .o-ui-lozenge.o-ui-lozenge-informative.o-ui-lozenge-highlight .o-ui-lozenge-text {
-    color: var(--o-ui-white);
+    color: inherit;
```
Comment (tjosepo) [inline packages/components/src/lozenge/src/Lozenge.css:57]: Do we need this property?

File: packages/components/src/lozenge/docs/Lozenge.stories.mdx [hunk 2]
```diff
@@ -19,7 +19,6 @@ import { Text } from "@components/typography";
     githubPath="/packages/components/src/lozenge/src"
 />
 
-
```
Comment (LaurentDario) [inline packages/components/src/lozenge/docs/Lozenge.stories.mdx:22]: Should we add a mention that changing the background or color of the Lozenge can lead to accessibility failures? Something along the lines of 
> Lozenge's variants follow the contrast guidelines. Changing the background or text color might cause contrast check failures.
(Not sure `contrast check failures` is the right way to put it)

**Review by LaurentDario — APPROVED** [review]
Just a documentation suggestion.

File: packages/components/src/lozenge/docs/Lozenge.stories.mdx (hunk 2 above)
Comment (wasim-abuzaher-sg) [reply packages/components/src/lozenge/docs/Lozenge.stories.mdx:22]: errors? issues? noncompliance?

Comment (fraincs) [reply packages/components/src/lozenge/docs/Lozenge.stories.mdx:22]: actually we don't want people to do it but we understand that we might need to do it and at least it won't fail. I will add it to the accessibility section of the lozenge doc, but we don't document it for other components like button / tag who also support changing the backgroundColor and color.

**Pull request merged**